dashboard/src/reducers.js

```diff
@@ -59,5 +59,8 @@ const makeRootReducer = () => (state, action) => {
     tutorial,
     unspent,
   })(state, action)
+  finalState.app = app(state.app, action, finalState.tutorial)
```

Comment:
I wonder if we can try to disentangle this state a little bit. It seems like the app reducer cares about state under the `app` key, but the `tutorial` key as well, which is a bit convoluted. Is there a reason why `tutorial` isn't a subkey under `app`?